SIS_import/sis_upload.rb

```diff
@@ -9,15 +9,21 @@
 
 ## make Canvas API GET call
 def Canvas_API_GET(url)
-	response = RestClient.get url, {:Authorization => "Bearer #{$token}",
+	begin
+		response = RestClient.get url, {:Authorization => "Bearer #{$token}",
 	                                :accept => "application/json",
 	                                :verify_ssl => true}
-	return JSON.parse(response)
+		return JSON.parse(response)
+	rescue => e
+		p "#{e.response} for #{url}"
```

Comment:
Still need the "p ..."?

Not all responses from the ESB will be JSON.  Some errors are in XML.  It may be better to address that later.